common/countries/Royal Fold.txt

```diff
@@ -0,0 +1,83 @@
+graphical_culture = UsGC
+color = { 50 50 224 }
```

Comment:
Make the colors slightly different -- you can use `debug color TAG r g b` to change a tag's color in game without actually having to reload every time

Reply:
What's the point? They don't border eachother, and the color is intended to show that they are DND's substates.

Reply:
Made Tharbad lighter